front/src/modules/simulationResult/components/SpaceTimeChart/SpaceTimeChart.tsx

```diff
@@ -218,7 +218,7 @@ export default function SpaceTimeChart(props: SpaceTimeChartProps) {
   /* coordinate the vertical cursors with other graphs (GEV for instance) */
   useEffect(() => {
     if (dateIsInRange(timePosition, timeScaleRange)) {
-      traceVerticalLine(chart, CHART_AXES.SPACE_TIME, positionValues, rotate, timePosition);
+      traceVerticalLine(chart, CHART_AXES.SPACE_TIME, positionValues, timePosition);
```

Comment:
You need to keep the rotate here, SpaceTimeChart is the only chart that can rotate (just change the order of parameters)
```suggestion
      traceVerticalLine(chart, CHART_AXES.SPACE_TIME, positionValues, timePosition, rotate);
```